example/example.cpp

```diff
@@ -24,507 +24,627 @@ using namespace isaac;
-					//Feedback for other clients than the changing one
-					if (rank == 0)
-						json_object_set_new( visualization->getJsonMetaRoot(), "interval", json_integer( interval ) );
-				}
-				json_decref( meta );
-			}
-			usleep(100);
-			count++;
-
-			//////////////////
-			// Debug output //
-			//////////////////
-			if (rank == 0)
-			{
-				int end = visualization->getTicksUs();
-				int diff = end-start;
-				if (diff >= 1000000)
-				{
-					merge_time -= kernel_time + copy_time;
-					printf("FPS: %.1f \n\tSimulation: %.1f ms\n\tDrawing: %.1f ms\n\t\tSorting: %.1f ms\n\t\tMerge: %.1f ms\n\t\tKernel: %.1f ms\n\t\tCopy: %.1f ms\n\t\tVideo: %.1f ms\n\t\tBuffer: %.1f ms\n",
-						(float)count*1000000.0f/(float)diff,
-						(float)full_simulation_time/1000.0f/(float)count,
-						(float)full_drawing_time/1000.0f/(float)count,
-						(float)sorting_time/1000.0f/(float)count,
-						(float)merge_time/1000.0f/(float)count,
-						(float)kernel_time/1000.0f/(float)count,
-						(float)copy_time/1000.0f/(float)count,
-						(float)video_send_time/1000.0f/(float)count,
-						(float)buffer_time/1000.0f/(float)count);
-					sorting_time = 0;
-					merge_time = 0;
-					kernel_time = 0;
-					copy_time = 0;
-					video_send_time = 0;
-					buffer_time = 0;
-					full_drawing_time = 0;
-					full_simulation_time = 0;
-					start = end;
-					count = 0;
-				}
-			}
-		}
-	}
-	MPI_Barrier(MPI_COMM_WORLD);
-	printf("%i finished\n",rank);
-
-	////////////////////
-	// Winter wrap up //
-	////////////////////
-	delete( visualization );
-
-	#if ISAAC_ALPAKA == 0
-		free(hostBuffer1);
-		free(hostBuffer2);
-		cudaFree(deviceBuffer1);
-		cudaFree(deviceBuffer2);
-	#endif
-
-	MPI_Finalize();
-	return 0;
+    isaac::DefaultCompositor
+#else
+    isaac::StereoController,
+#if (ISAAC_STEREO == 1)
+    isaac::StereoCompositorSideBySide<isaac::StereoController>
+#else
+    isaac::StereoCompositorAnaglyph<isaac::StereoController,0x000000FF,0x00FFFF00>
+#endif
+#endif
+    > (
+#if ISAAC_ALPAKA == 1
+        devHost, //Alpaka specific host dev instance
+        devAcc, //Alpaka specific accelerator dev instance
+        stream, //Alpaka specific stream instance
+#endif
+        name, //Name of the visualization shown to the client
+        0, //Master rank, which will opens the connection to the server
+        server, //Address of the server
+        port, //Inner port of the server
+        framebuffer_size, //Size of the rendered image
+        global_size, //Size of the whole volumen including all nodes
+        local_size, //Local size of the subvolume
+    {PARTICLE_VOLUME_X,PARTICLE_VOLUME_Y,PARTICLE_VOLUME_Z},
+    position, //Position of the subvolume in the globale volume
+    particle_sources,
+    sources, //instances of the sources to render
+    scaling
+    );
+
+    //Setting up the metadata description (only master, but however slaves could then add metadata, too, it would be merged)
+    if ( rank == 0 ) {
```

Comment:
Please revert this change `{` must be on an own line https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md#12-if-statements